### ISSUE TITLE
Crashing on video play - multi devices.

### ISSUE THREAD

**Issue by njappboy** [lbryio/odysee-roku#14]
Since yesterday I have been unable to play odysee  videos with this roku channel on multiple devices across 2 different locations. The device are a standalone roku attached to a television and a television with Roku bundles as part of the television model. When attempting to play any video the channel crashes returning to the Roku home screen.

**Comment by s9260** [comment]
In devel commit 2fb05a434e5592ccc5943c4ef1a11df5c2e1ed50 I fixed this, and [it is currently fixed in the testing/development version. You will have to sign into the Roku account that your Rokus use to add this unlisted channel](https://my.roku.com/account/add?channel=odyseeunofficialedge). The fixed version is scheduled to release on Monday.

Let me know if the development version doesn't work on your Roku TV, I've been having issues with certain models of TCL Roku TVs. Some work, others don't. I don't have the hardware to know why.

Why it broke:
I used to remove all special characters from the video URL because encodeuri wasn't working for the video player, and a temporary fix was done on cdn.lbryplayer.xyz. That fix may have been removed recently.
